tap_shopify/streams/transactions.py

```diff
@@ -107,19 +107,10 @@ def get_objects(self):
                 yield transaction
 
     def sync(self):
-        bookmark = self.get_bookmark()
-        max_bookmark = bookmark
         for transaction in self.get_objects():
             transaction_dict = transaction.to_dict()
-            replication_value = strptime_to_utc(transaction_dict[self.replication_key])
```

Comment:
Do provide an inline comment for this change.

Reply:
As we are now not filtering the transactions based on `created_at` after fetching all the transactions of updated orders. Hence we don't need a replication key for the transactions stream as it would be now incremental based on parent.